addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4246,6 +4246,9 @@ const SnippetOptionWidget = Widget.extend({
                 return;
             }
 
+            // GREP: PREVENT_DOUBLE_RELOAD
+            this.__willReload = requiresReload;
```

Comment:
Can't the save+reload technically fail? In which case you would want this to have been reset to false after the `_select` call so that some random calling `request_save` still works?